### PR TITLE
add support for 2.5.5 + debug

### PR DESCRIPTION
**Pull request by gdestuynder**

### CODE REVIEW

**Comment by gdestuynder** [comment]
@ameihm0912

**Comment by gdestuynder** [comment]
note: ive another PR to do after that to automate some bugzilla stuff

**Comment by gdestuynder** [comment]
meh just adding it to this PR :)

**Comment by gdestuynder** [comment]
And added auto assign support (demo: https://bugzilla.mozilla.org/show_bug.cgi?id=1361563)

how it works:
- set a pickle path (or use default) in the new options
- set an auto assign list (must be valid bugzilla emails) so that it has a default to load from

**Comment by claudijd** [comment]
r+

File: rra2json.inc.json [hunk 1]
```diff
@@ -15,7 +15,9 @@
 		"url": "https://bugzilla-dev.allizom.org/rest/",
 		"product": "",
 		"component": "",
-		"creator": ""
+		"creator": "",
+		"autoassign": [],
+		"cache": "/var/run/assignees.pickle"
```
Comment (ameihm0912) [inline rra2json.inc.json:20]: Does anything need to be added on the config side for these new items?

**Review by ameihm0912 — APPROVED** [review]

File: rra2json.inc.json (hunk 1 above)
Comment (ameihm0912) [reply rra2json.inc.json:20]: Nevermind, I just noticed your comment above 👍

Comment (gdestuynder) [reply rra2json.inc.json:20]: yeah, these are the ones mentioned in https://github.com/gdestuynder/rra2json/pull/5#issuecomment-298800194 

- you can keep the default for cache (or use something else if that path is not right)
- autoassign is a list of emails ["ameihm@mozilla.com", "tweir@mozilla.com", "jclaudius@mozilla.com", "gdestuynder@mozilla.com"]

**Pull request merged**